src/aap_eda/core/models/activation.py

```diff
@@ -74,6 +78,26 @@ class Meta:
     status_updated_at = models.DateTimeField(null=True)
     status_message = models.TextField(null=True, default=None)
 
+    def save(self, *args, **kwargs):
```

Comment:
It is missing the logic to ensure the status_message if you call "save" method without update_fields (only ensures the pending status) that's why I say you have to compare to the previous value, to check if the instance is updating that field or not.

Reply:
The missing logic is added